src/pages/students/[student_id].tsx

```diff
@@ -10,12 +10,14 @@ import $Image from "../../styles/Image.module.css";
 import $button from "@/components/design_system/button/Button.module.css";
 import $Form from "../../styles/Form.module.css";
 import $input from "@/styles/Input.module.css";
-import $Modal from "@/components/design_system/modal/Modal.module.css";
+import $CompassModal from "../../components/design_system/modal/CompassModal.module.css";
 import $StudentPage from "../../styles/StudentPage.module.css";
 
 import * as React from "react";
 import Typography from "@mui/material/Typography";
 
+// test comment
```

Comment:
Probably don't need this anymore?

Reply:
Thanks Connor! deleted this extraneous comment